services/web/src/pages/sign-up/index.page.jsx

```diff
@@ -29,11 +30,39 @@ const schema = yup.object().shape({
   ),
 });
 
+const passwordRules = [
+  {
+    title: 'Be a minimum of six characters',
+    done: false,
+  },
+  {
+    title: 'Have at least one letter',
+    done: false,
+  },
+  {
+    title: 'Have at least one number',
+    done: false,
+  },
+];
+
 const SignUp = () => {
   const [email, setEmail] = useState(null);
   const [registered, setRegistered] = useState(false);
   const [signupToken, setSignupToken] = useState();
 
+  const [passwordValue, setPasswordValue] = useState('');
+  const [passwordRulesData, setPasswordRulesData] = useState(passwordRules);
+
+  useEffect(() => {
+    const updatedPasswordRulesData = [...passwordRules];
+
+    updatedPasswordRulesData[0].done = passwordValue.length >= 6;
```

Comment:
You can avoid additional useState for password.
Use [getValues](https://react-hook-form.com/api/useform/getvalues) to get form's password value

Reply:
Shouldn't the [watch](https://react-hook-form.com/api/useform/watch) be used here?